libcontainer/cgroups/systemd/v2.go

```diff
@@ -198,6 +198,9 @@ func (m *unifiedManager) GetUnifiedPath() (string, error) {
 	m.mu.Lock()
 	defer m.mu.Unlock()
 	if m.path != "" {
+		if !cgroups.PathExists(m.path) {
```

Comment:
I would not do this in this function, but rather modify the callers to check that path exists.

Reply:
as this is a hot path and doing stat might not always be required.